lib/ccflay.rb

```diff
@@ -84,3 +84,25 @@ def pure_ruby_hash # :nodoc: see above
     hash
   end
 end
+
+class Sexp
+  def flatter
+    r = dup.clear
+
+    each do |s|
+      if s.is_a? Sexp
```

Comment:
:lipstick: Please use parens when invoking methods, as is our house style.

This applies throughout this block, but doesn't include `#<<`.